pkgs/development/python-modules/pynmea2/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "pynmea2";
+  version = "1.12.0";
+  name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "185wxn8gag9whxmysspbh8s7wn3sh1glgf508w2zzwi4lklryl7i";
+  };
+
+  # Tests are failing for Python 3 because the version is missing.
```

Comment:
Which version of what is missing? You can set `doCheck = isPy27`.

Reply:
I'm not quite sure, but the test fails with
```
_version (unittest.loader._FailedTest) ... ERROR

======================================================================
ERROR: _version (unittest.loader._FailedTest)
----------------------------------------------------------------------
AttributeError: module 'pynmea2' has no attribute '_version'

----------------------------------------------------------------------
```

Reply:
We can try reenabling the tests once the appropriate `checkPhase` is set.